Source/SwiftLintFramework/Rules/Style/VerticalWhitespaceClosingBracesRule.swift

```diff
@@ -8,121 +8,32 @@ private extension SwiftLintFile {
 
     public init() {}
 
-    private static let nonTriggeringExamples: [Example] = [
-        Example("[1, 2].map { $0 }.filter { true }"),
-        Example("[1, 2].map { $0 }.filter { num in true }"),
-        Example("""
-        /*
-            class X {
-
-                let x = 5
-
-            }
-        */
-        """)
-    ]
-
-    private static let violatingToValidExamples: [Example: Example] = [
-        Example("""
-        do {
-          print("x is 5")
-        ↓
-        }
-        """):
-            Example("""
-            do {
-              print("x is 5")
-            }
-            """),
-        Example("""
-        do {
-          print("x is 5")
-        ↓
-
-        }
-        """):
-            Example("""
-            do {
-              print("x is 5")
-            }
-            """),
-        Example("""
-        do {
-          print("x is 5")
-        ↓\n  \n}
-        """):
-            Example("""
-            do {
-              print("x is 5")
-            }
-            """),
-        Example("""
-        [
-        1,
-        2,
-        3
-        ↓
-        ]
-        """):
-            Example("""
-            [
-            1,
-            2,
-            3
-            ]
-            """),
-        Example("""
-        foo(
-            x: 5,
-            y:6
-        ↓
-        )
-        """):
-            Example("""
-            foo(
-                x: 5,
-                y:6
-            )
-            """),
-        Example("""
-        func foo() {
-          run(5) { x in
-            print(x)
-          }
-        ↓
-        }
-        """): Example("""
-            func foo() {
-              run(5) { x in
-                print(x)
-              }
-            }
-            """)
-    ]
-
     private let pattern = "((?:\\n[ \\t]*)+)(\\n[ \\t]*[)}\\]])"
+    private let trivialLinePattern = "((?:\\n[ \\t]*)+)(\\n[ \\t)}\\]]*$)"
 }
 
-extension VerticalWhitespaceClosingBracesRule: OptInRule, AutomaticTestableRule {
+extension VerticalWhitespaceClosingBracesRule: OptInRule {
     public var configurationDescription: String { return "N/A" }
 
-    public init(configuration: Any) throws {}
-
+    private static let examples = VerticalWhitespaceClosingBracesRuleExamples.self
     public static let description = RuleDescription(
         identifier: "vertical_whitespace_closing_braces",
         name: "Vertical Whitespace before Closing Braces",
         description: "Don't include vertical whitespace (empty line) before closing braces.",
         kind: .style,
-        nonTriggeringExamples: (violatingToValidExamples.values + nonTriggeringExamples).sorted(),
-        triggeringExamples: Array(violatingToValidExamples.keys).sorted(),
-        corrections: violatingToValidExamples.removingViolationMarkers()
+        nonTriggeringExamples: (examples.violatingToValidExamples.values +
+                                examples.nonTriggeringExamples).sorted(),
```

Comment:
Why `.sorted()`?

Reply:
I assumed this was the convention given the code that was previously present.
Removed (also in the next line).